src/helpers/format.ts

```diff
@@ -24,6 +24,6 @@ export function shortenString(str: string, length = 10) {
 export function parseIdentifier(identifier: string | null | undefined) {
   // replace non ascii chars
   return ethers.utils
-    .toUtf8String(identifier || "")
+    .toUtf8String(identifier || [])
```

Comment:
apparently empty string causes errors, empty array is better since this expects a byte like thing